src/Components/Details.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '../CSS_styling/Details.css';
+
+const Details = ({movieInfo}) => {
+  let detail = movieInfo.map((movie) => {
+    return (
+        <div className='details-div'>
```

Comment:
I think you should be able to add the key to the div here like `<div key={movie.id} className='details-dive, etc>`